data-model/src/main/avro/eventfields/jaeger/JaegerFields.avdl

```diff
@@ -4,6 +4,5 @@ protocol JaegerFieldsProtocol {
     int flags = 0;
     union { null, array<string> } logs = null;
     union { null, array<string> } warnings = null;
-    union { null, string } service_name = null;
```

Comment:
Removing the ```service_name``` . We looked into usages, everywhere it is being consumed using the constant 
```JaegerAttribute.JAEGER_ATTRIBUTE_SERVICE_NAME``` . Setter is being used in ```JaegerSpanNormaliser``` in repo ```span-normaliser``` which will be removed.

I'm assuming it's a action. 
cc: @buchi-busireddy @kotharironak